code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -441,6 +441,11 @@
 	orient2hud()
 	return
 
+/obj/item/weapon/storage/Destroy()
+	qdel(boxes)
+	qdel(closer)
+	return ..()

```

Comment:
I kinda feel like Destroying a storage item should have it either eject its contents or qdel them... though we should probably just have *any* movable that doesn't handle its contents qdel all of them.